README.md

```diff
@@ -55,11 +55,14 @@ Let's face it, you are probably planning to use this plugin the same way as many
 Several example profiles are included below. Run the corresponding command snippet to install each profile.
 
 - `mainstall OutpostAirwolf` -- Adds an Air Wolf vendor to Outpost, with some ladders to allow access.
+- `mainstall OutpostExtended` -- Adds bandtit wheel, slot machines and poker tables to Outpost.
```

Comment:
We should also mention that this profile also adds an Air Wolf vendor. One issue with that is that somebody might install it and OutpostAirwolf, and therefore end up with 2 vendors and spawn points. Maybe we could add a note here indicating that people should only one install one of these profiles. Also, there's a typo: "bandtit".

Reply:
Fixed typo and updated description. What about removing Air Wolf from OutpostExtended profile so it contains only gambling?

Reply:
I agree with generally keeping profiles as modular as possible. However, this is a tricky case because both OutpostAirwolf and OutpostExtended need ladders to allow access to rooftops. I want to avoid having double ladders in the same place. There are several ways we could address that problem.

- We could add separate ladders in a different location, specifically in the OutpostExtended profile.
- We could add logic to avoid double spawning entities, as detected by guid or prefab+position+rotation. However, this would significantly increase complexity in ways that make me dizzy.
- We could introduce a mechanism for profile dependencies, where one profile requires another to be installed to function. This also introduces complexity which I don't think we need yet. Note: Implicity, there's already a dependency concept today, where profiles may depend on other plugins (i.e., Sign Artist, Entity Scale Manager, Custom Vending Setup, Copy Paste) in order to function as designed, for which there is currently no mechanism to warn the user. I see this as being an existing area for improvement and probably unrelated to profile dependencies, but it might be something that we can solve under the same umbrella when the time comes.
- We could add validation logic to prevent loading a profile (or to add a required confirmation to load a profile) that is incompatible with another. Compatibility could be determined several ways, such as by duplicate guids (we can ensure that the guids are duplicated in the profiles we publish), duplicate prefab+position+rotation, or by an explicit declaration like `"IncompatibleProfiles": ["OutpostAirWolf"]`. As an alternative to blocking the user, we could also just add a warning after the fact.

For now, considering that we control all of the profiles that get published in this repo, we should strive to make each profile independent and compatible, to avoid needing to add complexity to the plugin's implementation. While it might be an interesting problem to solve profile compatibility, I think we have much bigger fish to fry at this time. If we are unable to make them compatible, documenting as you have done in this revision is a reasonable initial mitigation.

At this point, I recommend two options:
- Simply declare the incompatibility in the documentation as you have done already
- Don't document the OutpostExtended profile until we have had more time to think about this problem

Reply:
It was foolish to assume that you just forgot add OutpostExtended to docs lol. Since scope of this PR is to add 2 profiles, I removed OutpostExtended from docs again, rather than intruduce compatibility issues. 

Another easy solution would be simply to move OutpostAirwolf to different roof, therefore dodging all related issues with ladders. But since (almost) everybody using in that exact roof, players could be confused. I vote for OutpostExtended ladder rework